msbuild/Xamarin.Localization.MSBuild/xlf/MSBStrings.cs.xlf

```diff
@@ -0,0 +1,1943 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Curious - I assume this file is autogenerated?

Reply:
Yes, all `.xlf` are autogenerated.